lzy/whiteboard/src/main/java/ai/lzy/whiteboard/storage/WhiteboardStorage.java

```diff
@@ -13,15 +12,9 @@
 
 public interface WhiteboardStorage {
 
-    void insertWhiteboard(String userId, Whiteboard whiteboard,
+    void insertWhiteboard(String userId, Whiteboard whiteboard, Instant ts,
```

Comment:
why `insert`?
I'd expect to have `createWhiteboard` method here

Reply:
Actually, whiteboard is not created here, it is just added. Maybe `add` is better word